src/components/ErrorContent/ErrorContent.module.scss

```diff
@@ -1,33 +1,54 @@
-@import '../../../node_modules/@santiment-network/ui/mixins';
+@import "../../../node_modules/@santiment-network/ui/mixins";
 
 .container {
   display: flex;
-  max-width: 1152px;
-  margin: auto;
   align-items: center;
+  justify-content: center;
+  flex: 1;
+  gap: 40px;
+}
 
-  @include responsive('phone', 'phone-xs', 'tablet', 'laptop') {
-    flex-direction: column-reverse;
-    padding: 16px;
-  }
+:global(.laptop) .container,
+:global(.phone) .container,
+:global(.tablet) .container,
+:global(.phone-xs) .container {
+  flex-direction: column-reverse;
+  justify-content: flex-end;
+  gap: 0;
+  padding: 16px 16px 80px;
+}
+
+:global(.desktop) .bigImgContainer {
+  justify-content: space-between;
+  padding-left: 10%;
+}
+
+:global(.desktop) .info {
+  min-width: 400px;
 }
 
 .title {
-  @include text('h2', 'l');
+  @include text("h2", "l");
+}
 
-  @include responsive('phone', 'phone-xs', 'tablet', 'laptop') {
-    @include text('h4', 'l');
-  }
+:global(.laptop) .title,
+:global(.phone) .title,
+:global(.tablet) .title,
+:global(.phone-xs) .title {
+  @include text("h4", "l");
 }
```

Comment:
`[D-0-2]` Following case can be reduced to `.title` and `:global(.desktop) .title` rules, since in current implementation `@include text("h4", "l");` is changed for everything but desktop. The result might look like this:
```scss
title {
   @include text("h4", "l");

   :global(.desktop) & {
      @include text("h2", "l");
   }
 }
```